presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ColumnValidator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.verifier.checksum;
+
+import com.facebook.presto.sql.tree.Identifier;
+import com.facebook.presto.sql.tree.SingleColumn;
+import com.facebook.presto.verifier.framework.Column;
+
+import java.util.List;
+
+abstract class ColumnValidator
+{
+    abstract List<SingleColumn> generateChecksumColumns(Column column);
+
+    abstract boolean isMatched(Column column, ChecksumResult controlResult, ChecksumResult testResult);
+
+    static Identifier identifier(String name)
```

Comment:
Is this function the reason this is an abstract class rather than an interface? Does it make more sense to have a `getIdentifier` function in the `Column` class instead?

Reply:
Good catch. Yes, I'm converting this to an interface and move the static method into `VerifierUtil#delimitedIdentifier`. Also adding `Column#getIdentifier`.